src/fides/api/ctl/routes/system.py

```diff
@@ -88,6 +88,14 @@ async def validate_privacy_declarations_data_uses(
                 status_code=HTTP_400_BAD_REQUEST,
                 detail=f"Invalid privacy declaration referencing unknown DataUse {privacy_declaration.data_use}",
             )
+        logical_id = privacy_declaration_logical_id(privacy_declaration)
+        if logical_id in logical_ids:
```

Comment:
If we're looking at `logical_ids` to determine data use uniqueness then it's possible at least from the backend that two privacy declarations could be added with different names but the same data uses, causing the same data use to be added twice.

Reply:
that's right, and my understanding is that is actually desired behavior. we want the `name` (which shows up as `processing activity` in the UI), to be a mechanism that user's can leverage to create >1 privacy declarations on the same system with the same data use.  my understanding is that it's an edge case, but it's one we want/need to continue supporting at this time.